core/trino-main/src/test/java/io/trino/operator/output/BenchmarkPartitionedOutputOperator.java

```diff
@@ -323,7 +327,34 @@ public OptionalInt getNullChannel()
             MAP_BIGINT_BIGINT(createMapType(BigintType.BIGINT, BigintType.BIGINT), 1000),
             MAP_BIGINT_MAP_BIGINT_BIGINT(createMapType(BigintType.BIGINT, createMapType(BigintType.BIGINT, BigintType.BIGINT)), 1000),
             ROW_BIGINT_BIGINT(rowTypeWithDefaultFieldNames(ImmutableList.of(BigintType.BIGINT, BigintType.BIGINT)), 1000),
-            ROW_ARRAY_BIGINT_ARRAY_BIGINT(rowTypeWithDefaultFieldNames(ImmutableList.of(new ArrayType(BigintType.BIGINT), new ArrayType(BigintType.BIGINT))), 1000);
+            ROW_ARRAY_BIGINT_ARRAY_BIGINT(rowTypeWithDefaultFieldNames(ImmutableList.of(new ArrayType(BigintType.BIGINT), new ArrayType(BigintType.BIGINT))), 1000),
+            ROW_RLE_BIGINT_BIGINT(rowTypeWithDefaultFieldNames(ImmutableList.of(BigintType.BIGINT, BigintType.BIGINT)), 1000) {
+                @Override
+                public Page createPage(List<Type> types, int positionCount, float nullRate)
+                {
+                    return createRandomPage(types, positionCount, Optional.of(ImmutableList.of(0)), types.stream()
+                            .map(type -> {
+                                boolean[] isNull = null;
+                                int nullPositionCount = 0;
+                                if (nullRate > 0) {
+                                    isNull = new boolean[positionCount];
+                                    Set<Integer> nullPositions = chooseNullPositions(positionCount, nullRate);
```

Comment:
no need to iterate over array
```
Set<Integer> nullPositions = chooseNullPositions(positionCount, nullRate);
nullPositions.stream()
  .forEach(position -> isNull[position] = true);
nullPositionCount = nullPositions.size()
```

Reply:
`isNull` is not final so cannot use lambda but changed it to iterate over `nullPositions`